playground/TestShop/TestShop.AppHost/Properties/launchSettings.json

```diff
@@ -25,7 +24,6 @@
         "ASPIRE_DASHBOARD_OTLP_ENDPOINT_URL": "http://localhost:16031",
         //"ASPIRE_DASHBOARD_OTLP_HTTP_ENDPOINT_URL": "http://localhost:16032",
         "ASPIRE_RESOURCE_SERVICE_ENDPOINT_URL": "http://localhost:17031",
-        "ASPIRE_SHOW_DASHBOARD_RESOURCES": "true",
```

Comment:
?

Reply:
https://github.com/dotnet/aspire/issues/3936

Reply:
I think the solution is this change. Hide it by default.

Reply:
The solution is not to do per request logging